jekyll/_cci2/non-aws.md

```diff
@@ -86,49 +86,49 @@ The following sections describe the steps for installation of the Services VM an
 
 ### Prerequisites
 
-Have the following information available before beginning the installation procedure:
+Have the following information/resources available before beginning the installation procedure:
 
-- CircleCI License file (.rli), contact CircleCI support for a license.
-- A machine to run Ubuntu 14.04 with a minimum of at least 100 GB storage, 32 GB RAM, and 4 CPUs (8 CPUs preferred) for the Services VM.
-- A cluster of machines running Ubuntu 14.04 with minumum of 8 GB RAM, 4 CPUs each, and network access any Docker registries that are required by your builds for the Nomad Client VMs.
+- CircleCI License file (.rli)—contact CircleCI support for a license
+- A machine to run Ubuntu 14.04 with a minimum of at least 100 GB storage, 32 GB RAM, and 4 CPUs (8 CPUs preferred) for the Services VM
+- A cluster of machines running Ubuntu 14.04 with a minumum of 8 GB RAM and 4 CPUs each, as well as network access to any Docker registries that are required by your builds for the Nomad Client VMs
 
 ### Steps
 
-To install the Services machine, perform the following steps:
+**To install the Services machine, perform the following steps:**
 
 1. Copy the [Services init script](https://github.com/circleci/server-static-install/blob/master/provision-services-ubuntu.sh) to the Services VM machine.
 
 2. Log in to the machine provisioned for the Services VM and run the `sudo su` command.
 
-3. Run './provision-services-ubuntu.sh' to start the script. 
+3. Run `./provision-services-ubuntu.sh` to start the script. 
 
 4. Go to the public IP of the host on port 8800 using HTTPS.
 
 5. Enter your license.
 
-6. Complete the Storage section. If you are not using a cloud service , then you will pick `None`. More information below.
+6. Complete the Storage section. If you are not using a cloud service, then you will pick `None`—more information below.
 
 7. Set the VM Provider to None.
 
 8. Set 1.0 Builds to 'Off'.
 
 9. Set 2.0 Builds to 'Clustered'.
 
-To install the Nomad Clients, perform the following steps:
+**To install the Nomad Clients, perform the following steps:**
```

Comment:
We can use heading for both of these to make it more obvious on the page. Just add them above this sentence 
### Installing Nomad Clients